dart/lib/src/sentry_options.dart

```diff
@@ -186,10 +186,12 @@ class SentryOptions {
 
   /// The ignoreErrors tells the SDK which errors should be not sent to the sentry server.
   /// If an null or an empty list is used, the SDK will send all transactions.
+  /// To use regex add the `^` and the `$` to the string.
   List<String> ignoreErrors = [];
 
   /// The ignoreTransactions tells the SDK which transactions should be not sent to the sentry server.
   /// If null or an empty list is used, the SDK will send all transactions.
+  /// To use regex add the `^` and the `$` to the string.
```

Comment:
this implies that if you don't add these characters, regex won't be used. That is not the case from what I can tell - to me it looks like we always use regex

Reply:
yes, you are right. I will change this.